solutions/recursion/jumping-game.js

```diff
@@ -0,0 +1,33 @@
+let fs = require("fs");
+let data = fs.readFileSync(0, 'utf-8');
+let idx = 0;
+data = data.split('\n');
+
+function readLine() {
+	idx++;
+	return data[idx - 1];
+}
+
+// -------- Do NOT edit anything above this line ----------
+// Use readLine() for taking input, it will read one line of from the input  and is stored in string format
+
+function minimumJumps(currentPosition, panels, numberOfPanels){
+
+    if(currentPosition > numberOfPanels){
+        return 0;
+    }
+
+    return 1 + Math.min(minimumJumps(currentPosition + panels[currentPosition - 1], panels, numberOfPanels), 
+		minimumJumps(currentPosition + 1, panels, numberOfPanels));
```

Comment:
@prateek-mehra , please store the the result of  `minimumJumps(currentPosition + panels[currentPosition - 1], panels, numberOfPanels)` in a variable and the result of `minimumJumps(currentPosition + 1, panels, numberOfPanels)` in another variable and use those variables in return statement

Reply:
@LVSsandeepVarma Done.